lib/webhooks/webhooks.js

```diff
@@ -52,14 +80,24 @@ function validateExpressRequest(request, authToken, opts) {
     if (request.originalUrl.search(/\?/) >= 0) {
       webhookUrl = webhookUrl.replace("%3F","?");
     }
+
   }
 
-  return validateRequest(
-    authToken,
-    request.header('X-Twilio-Signature'),
-    webhookUrl,
-    request.body || {}
-  );
+  if (webhookUrl.indexOf('bodySHA256') > 0) {
+    return validateRequestWithBody(
+      authToken,
+      request.header('X-Twilio-Signature'),
+      webhookUrl,
+      request.body || {}
+    );
+  } else {
+    return validateRequest(
+      authToken,
+      request.header('X-Twilio-Signature'),
+      webhookUrl,
+      request.body || {}
```

Comment:
wait does `request.body` contain the params?

Reply:
Yup, same name. In express, `request.body` is auto-converted based on the content-type.

I think that's your question, right?